.buildkite/pipeline.yml

```diff
@@ -33,250 +33,82 @@ x-common-params:
-    if: *is_branch_for_quick_ui_tests
-    depends_on:
-      - ios-build-and-saucelabs
-      - run-full-ui-test
-    command: .buildkite/commands/test-ios.sh
-    plugins:
-      - *ci_toolkit_plugin
-      - *nvm_plugin
-      - *git-cache-plugin
-    artifact_paths:
-      - reports/test-results/ios-test-results.xml
-    agents:
-      queue: mac
-    env: *xcode_agent_env
-
-  # Same step as above, but will always run in trunk, release/, and dependabot/submodules branches
-  - label: Test iOS on Device – Full iPhone
-    # The full UI tests suite version depends only on the ios-build step, meaning it has no manual step that triggers it
-    if: *is_branch_for_full_ui_tests
-    depends_on:
-      - ios-build-and-saucelabs
-    command: .buildkite/commands/test-ios.sh
-    plugins:
-      - *ci_toolkit_plugin
-      - *nvm_plugin
-      - *git-cache-plugin
-    artifact_paths:
-      - reports/test-results/ios-test-results.xml
-    agents:
-      queue: mac
-    env: *xcode_agent_env
+      - reports/test-results/android-test-results.xml
+    env:
+      JEST_JUNIT_OUTPUT_FILE: reports/test-results/android-test-results.xml
+    # notify:
+    #   - github_commit_status:
+    #       context: Test Android on Device - Canaries
 
-  - label: Test iOS on Device – Full iPad
-    # The quick UI tests suite version depends on the block step being unblocked
-    if: *is_branch_for_quick_ui_tests
-    depends_on:
-      - ios-build-and-saucelabs
-      - run-full-ui-test
-    command: .buildkite/commands/test-ios.sh --ipad
+  - label: Test Android on Device – Canary Pages (Linux)
+    depends_on: android-build-and-saucelabs
+    command: .buildkite/commands/test-android.sh --canary
     plugins:
       - *ci_toolkit_plugin
       - *nvm_plugin
       - *git-cache-plugin
     artifact_paths:
-      - reports/test-results/ios-test-results.xml
-    agents:
-      queue: mac
-    env: *xcode_agent_env
+      - reports/test-results/android-test-results.xml
+    env:
+      JEST_JUNIT_OUTPUT_FILE: reports/test-results/android-test-results.xml
+    # notify:
+    #   - github_commit_status:
+    #       context: Test Android on Device - Canaries
 
-  - label: Test iOS on Device – Full iPad
-    # The full UI tests suite version depends only on the ios-build step, meaning it has no manual step that triggers it
-    if: *is_branch_for_full_ui_tests
-    depends_on:
-      - ios-build-and-saucelabs
-    command: .buildkite/commands/test-ios.sh --ipad
+  - label: Test Android on Device – Canary Pages (macOS)
+    depends_on: android-build-and-saucelabs
+    command: .buildkite/commands/test-android.sh --canary
     plugins:
       - *ci_toolkit_plugin
       - *nvm_plugin
       - *git-cache-plugin
     artifact_paths:
-      - reports/test-results/ios-test-results.xml
+      - reports/test-results/android-test-results.xml
     agents:
-      queue: mac
-    env: *xcode_agent_env
+      queue: macos
+    env:
+      <<: *xcode_agent_env
```

Comment:
The build here staid waiting for an agent for more than a week 🤔 

~~I clearly misconfigured something, but I can't see what...~~ Just as I wrote that, I realized it: `queue: macos` instead of the correct `queue: mac` 🤦‍♂️